### PR TITLE
Add missing alt text to images in documentation and fix broken images

### PR DESCRIPTION
**Pull request by IanSkelskey**
This pull request adds alt text to images in various documentation files that were missing alt text and fixes a few broken images. This effort is aimed at improving accessibility and ensuring compliance with web accessibility standards. Special thanks to @sandbergja for running vale and providing the report on the missing alt tags. It would be a good practice to block pull requests with a tool like this to make sure style rules are applied on the first go.

As a new contributor to the Evergreen source project and a new Evergreen Systems Specialist, I have little experience with libraries but am familiar with software. I appreciate a thorough review of my contributions to ensure accuracy and completeness.

Thank you for your guidance and support.

## Changes

- Added alt text to images in multiple documentation files.
- Fixed typo in `docs/modules/admin/pages/emergency_closing_handler.adoc` that affected 6 images.

### Notes

> There are some commented out images that I noticed in `docs/modules/circulation/pages/circulating_items_web_client.adoc` on the following lines:
>
> - 216, 838, 854, 858
>
> Not sure if there is a reason for this, but I wanted to point it out.

> I'm not exactly sure how to interpret the images found in `docs/modules/cataloging/pages/MARC_Editor.adoc`. Someone more experienced with the software/libraries may be better suited for providing the alt text here.

### Files changed

- docs/modules/admin/pages/MARC_RAD_MVF_CRA.adoc
- docs/modules/admin/pages/booking-admin.adoc
- docs/modules/admin/pages/closed_dates.adoc
- docs/modules/admin/pages/emergency_closing_handler.adoc
- docs/modules/admin/pages/web_client-login.adoc
- docs/modules/admin/pages/z3950_servers.adoc
- docs/modules/admin/pages/virtual_index_defs.adoc
- docs/modules/admin/pages/latency.adoc
- docs/modules/admin/pages/multilingual_search.adoc
- docs/modules/serials/pages/F-Special_issue.adoc
- docs/modules/serials/pages/B-serials_admin.adoc
- docs/modules/serials/pages/E-routing_lists.adoc
- docs/modules/serials/pages/C-serials_workflow.adoc
- docs/modules/serials/pages/H-holdings_statements.adoc
- docs/modules/serials/pages/D-Receiving.adoc
- docs/modules/cataloging/pages/MARC_Editor.adoc
- docs/modules/cataloging/pages/record_buckets.adoc
- docs/modules/cataloging/pages/volcopy_editor.adoc
- docs/modules/cataloging/pages/overlay_record_3950_import.adoc
- docs/modules/opac/pages/tpac_meta_record_holds.adoc
- docs/modules/circulation/pages/booking.adoc
- docs/modules/circulation/pages/basic_holds.adoc
- docs/modules/circulation/pages/circulating_items_web_client.adoc

### CODE REVIEW

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks so much for this huge, detailed effort @IanSkelskey -- it's a big improvement for accessibility in the manual!

If you want to check out what the manual looks like with your changes, [Github Actions generated a copy here](https://github.com/evergreen-library-system/Evergreen/suites/25969335952/artifacts/1698280687) -- there is also a `docs/generate_docs.pl` script that you can run locally if you'd like, which is the same thing that runs on the docs.evergreen-ils.org server.

Overall, this looks great.  The only real blocker I see is an annoying asciidoc syntax issue: [if your alt text contains commas, you need to add some double quotes around the text](https://docs.asciidoctor.org/asciidoc/latest/macros/images/#block-image-macro).  Without these, the alt text gets truncated, and the values after the comma get tossed into the image's width and height attributes.  For example, in the generated docs, if you inspect the first image in docs/latest/admin/booking-admin.html, it's:

```
<img src="_images/booking/booking-create-resourcetype_webclient-1.png" alt="Options in the Booking Administration menu which include Resources" width="Resource Attribute Values" height="Resource Types">
```

File: docs/modules/admin_initial_setup/pages/borrowing_items.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ identify the maximum renewals that can be placed on an item.
 You can find Circulation Duration Rules by navigating to *Administration
 -> Server Administration -> Circulation Duration Rules*. 
 
-image::borrowing_items/circ_duration_rules.jpg[]
+image::borrowing_items/circ_duration_rules.jpg[Table titled 'Circ Duration Rules' showing circulation duration rules. Columns: #: row number, name: rule name, max_renewals, shrt: short duration, normal: normal duration, extended: extended duration. Example rows: 21d_1r (1 renewal, 21 days for all durations), 14d_0r (0 renewals, 14 days for all durations).]
```
[the remark this answers is not in the document]
Comment (IanSkelskey) [reply docs/modules/admin_initial_setup/pages/borrowing_items.adoc:75]: Agreed.

File: docs/modules/admin/pages/booking-admin.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@ You need to create resource types and resource attributes (features of the resou
 
 1) Select Administration -> Booking Administration -> Resource Types.
 
-image::booking/booking-create-resourcetype_webclient-1.png[]
+image::booking/booking-create-resourcetype_webclient-1.png[Options in the Booking Administration menu which include Resources, Resource Attribute Values, Resource Types, Resource Attribute Maps, and Resource Attributes.]
```
[the remark this answers is not in the document]
Comment (IanSkelskey) [reply docs/modules/admin/pages/booking-admin.adoc:22]: I can fix that. Thank you.

[the remark this answers is not in the document]
Comment (IanSkelskey) [reply docs/modules/admin/pages/booking-admin.adoc]: @sandbergja I think I got them all now.

[the remark this answers is not in the document]
Comment (IanSkelskey) [reply docs/modules/admin/pages/booking-admin.adoc]: Also another good candidate for a style rule. Perhaps the default should be that all alt tags are wrapped in double quotes.

**Review by sandbergja — APPROVED** [review]
Looks great, thanks @IanSkelskey

**Comment by sandbergja** [comment]
Thanks, @IanSkelskey !  I've squashed your commits into one and pushed them to:

- main
- the rel_3_11, rel_3_12, and rel_3_13 release branches

I don't remember how often the docs rebuild, but you should definitely see your changes reflected at https://docs.evergreen-ils.org within 24 hours.

**Pull request closed without merging**